examples/deepcopy-gen/output_tests/maps/zz_generated.go

```diff
@@ -100,6 +100,18 @@ func (in *Ttest) DeepCopyInto(out *Ttest) {
 			(*out)[key] = val
 		}
 	}
+	if in.StringPtr != nil {
+		in, out := &in.StringPtr, &out.StringPtr
+		*out = make(map[string]*string, len(*in))
+		for key, val := range *in {
+			if val == nil {
+				(*out)[key] = nil
+			} else {
+				(*out)[key] = new(string)
+				*(*out)[key] = *val
```

Comment:
Unless go manages to optimize it, this is a lot of map deference operations?

```go
x := new(string)
*x = *val
(*out)[key] = x
```

might be fewer?

Quite probably, not worth changing--up to you.

Reply:
might be easier to read

Reply:
will optimize that.

Reply:
fixed